src/libslic3r/PrintConfig.cpp

```diff
@@ -2544,9 +2544,9 @@ void PrintConfigDef::init_fff_params()
     def->set_default_value(new ConfigOptionStrings { " " });
 
     def = this->add("single_extruder_multi_material", coBool);
-    //def->label = L("Single Extruder Multi Material");
-    //def->tooltip = L("Use single nozzle to print multi filament");
-    def->mode = comDevelop;
+    def->label = L("Single Extruder Multi Material");
+    def->tooltip = L("Use single nozzle to print multi filament");
+    def->mode = comAdvanced;
```

Comment:
It's better to keep this option hidden as of now. 
The MMU logic has changed dramatically in BS. 

Also better not to include this change in this PR.

Reply:
sorry, this is a leftover for something else I'm working on...